gabrielgomesml/training_project_client
# Feature/Add-Movie

## What I did

- Add movie page

## How to test

- First of all make sure you are running the most updated server version in your machine. You can get it here: https://github.com/gabrielgomesml/training_project_server/pull/2
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Log into your account and you'll be redirect to the homepage;
- Click in the "+" button to be redirect to Add Movies page;
- Fill the forms and try to add an movie;
- Check in your database if the movie was created and your account is related to them;
